src/pages/_app.tsx

```diff
@@ -14,6 +14,11 @@ import { Fragment, useEffect, useState } from "react"
 import { SWRConfig } from "swr"
 import "theme/custom-scrollbar.css"
 import fetcher from "utils/fetcher"
+/**
+ * Polyfill HTML inert property for Firefox support:
+ * https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inert#browser_compatibility
+ */
+require("wicg-inert")
```

Comment:
Could we use a simple `import` here (just for consistency)

Reply:
Oh right, I've only tried `import wicg-inert from "wicg-inert"` and that got removed by VS Code because of being unused, so went with a require. But simply `import "wicg-inert"` works, changed it, thanks!